### ISSUE TITLE
Only show the welcome page the first time the extension starts

### ISSUE THREAD

**Issue by fbricon** [redhat-developer/vscode-granite#7]
We need to check and store a flag in the global storage, so the welcome page only appear the 1st time on startup

**Comment by lavanyaj3** [comment]
Before installing ollama I'm getting this as the welcome page
<img width="1268" alt="image" src="https://github.com/user-attachments/assets/1de2224e-3e3b-4cdd-867c-6a0adf9457b0">
After installing ollama am getting the below welcome page
<img width="1669" alt="image" src="https://github.com/user-attachments/assets/0f87b6af-93e3-4a7b-b2c7-fd224bbed322">

**Comment by lavanyaj3** [comment]
@fbricon request you to provide more explanation on this

**Comment by fbricon** [comment]
In extension.ts#activate, we need something like 
```
  const hasRunBefore = context.globalState.get('hasRunSetup', false);
  if (!hasRunBefore) {
    await context.globalState.update('hasRunSetup', true);
    return commands.executeCommand('vscode-granite.setup');
  }
  ```
  
  Could also test against an environment variable set in the launch config, like OLLAMA_MOCK, to ensure it always starts when running the extension host in dev mode

**Comment by fbricon** [comment]
fixed with #56
